spec/features/admins/merchant_downgrade_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'role downgrade' do
+  it 'admin-users can downgrade the role of merchants' do
+
+    admin = create(:admin)
+    merchant = create(:merchant)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+    visit "/admin/merchants/#{merchant.id}"
+    click_on 'downgrade to user'
+    expect(current_path).to eq(admin_user_path(merchant.id))
```

Comment:
Should have test where there is content only related to user and second test with flash message indicating downgrade and third requirement that merchant info is not present